src/main.cpp

```diff
@@ -112,6 +111,21 @@ void os_getDevEui (u1_t* buf) {
         gen_lora_deveui(buf); // generate DEVEUI from device's MAC
 }
 
+// LMIC enhanced Pin mapping
+const lmic_pinmap lmic_pins = {

```

Comment:
PlatformIO reports "Identifier undefined" for the lmic_pins, this was a problem in master version before already. Don't know how to solve this.